rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -276,20 +253,19 @@ class ROCALClassificationIterator(ROCALGenericIterator):
     fill_last_batch = True, last_batch_padded = True   -> last batch = [7, 7], next iteration will return [1, 2]
     fill_last_batch = True, last_batch_padded = False  -> last batch = [7, 1], next iteration will return [2, 3]
     """
+
     def __init__(self,
                  pipelines,
+                 device = "cpu",
+                 device_id = 0,
                  size = 0,
-                 auto_reset=False,
-                 fill_last_batch=True,
-                 dynamic_shape=False,
-                 last_batch_padded=False,
-                 display=False,
-                 device="cpu",
-                 device_id =0):
+                 auto_reset = False,
+                 fill_last_batch = True,
+                 dynamic_shape = False,
+                 last_batch_padded = False):
         pipe = pipelines
         super(ROCALClassificationIterator, self).__init__(pipe, tensor_layout = pipe._tensor_layout, tensor_dtype = pipe._tensor_dtype,
-                                                            multiplier=pipe._multiplier, offset=pipe._offset,display=display, device=device, device_id = device_id)
-
+                                                          multiplier = pipe._multiplier, offset = pipe._offset, device = device, device_id = device_id)
```

Comment:
Revert changes in this line, display is removed?

Reply:
rverted, display was removed , yes